apps/application-system/api/src/app/app.module.ts

```diff
@@ -56,6 +57,7 @@ import { JudicialAdministrationClientConfig } from '@island.is/clients/judicial-
         ChargeFjsV2ClientConfig,
         PassportsClientConfig,
         PaymentScheduleClientConfig,
+        EhicClientConfig,
```

Comment:
Need to add required env variables to the application-system-api infra configuration (check out the infra folder in this app).

Reply:
I have added ```
 EHIC_XROAD_PROVIDER_ID:
        '/k8s/application-system-api/EHIC_XROAD_PROVIDER_ID',
```
In the application-system-api.ts
Please see it this solves this issue.

Reply:
Did you create this secret in all environments with correct values?

Usually we have provider ids as [plain text variables](https://github.com/island-is/island.is/blob/eda3cf89ea4e421bf55360d55d09119c2615462e/infra/src/dsl/xroad.ts#L178-L182).

Reply:
This is my first time doing this. I don't know these values and I don't know who would.
 Who could help me with this? I'm not sure I put this in the right section in this file. 
 There is just the path that would be an env variable so did I even put this in the right section: Secrets?